### ISSUE TITLE
Compat formats will be killed off

### ISSUE THREAD

**Issue by myrsloik** [Endilll/vapoursynth-preview#31]
The compat formats will be removed in future builds of VS meaning that you'll have to use RGB24 as output and pack the image yourself. You can get the beta builds with this behavior here:
https://forum.doom9.org/showthread.php?t=183070

**Comment by Endilll** [comment]
Thank you for letting me know.
Can Python API in those beta builds be considered stable? Can you give any estimates if it's not?
Again, any estimates for proper release with APIv4?
I'm interested not just in compat formats removed, but in audio support as well.

**Comment by myrsloik** [comment]
> Can Python API in those beta builds be considered stable?
Yes, there are no further changes planned. Overall there are very few changes on the Python side apart from the minimum necessary due to core functionality differences.

> Again, any estimates for proper release with APIv4?
A few weeks to a month I guess. Very few bugs reported so far and scripts/applications seem to get updated reasonably fast.

**Comment by Endilll** [comment]
Good news, thank you. I'll make sure to update vapoursynth-preview, and revisit audio support proof-of-concept I made against some early builds year and a half ago.

**Comment by shssoichiro** [comment]
Do you have an update on this? The breaking changes to vapoursynth have been out for a few weeks and are now pushed to the Arch repos. vspreview is completely broken with the new version.

**Comment by DJATOM** [comment]
@shssoichiro As temporary workaround you may use my [patch ](https://github.com/DJATOM/vapoursynth-preview/commit/4e7c73597df3576d6cd374a16bfeaf3f7f0bea86) (with [libp2p](https://github.com/DJATOM/LibP2P-Vapoursynth) plug-in).

**Comment by shssoichiro** [comment]
I'm still receiving the same error with that patch. To be specific, the error I'm getting is `module 'vapoursynth' has no attribute 'AlphaOutputTuple'` in core/types.py line 697.

**Comment by shssoichiro** [comment]
I'm trying to do some debugging locally. Looks like `AlphaOutputTuple` was renamed to `VideoOutputTuple`, but after fixing that, I hit another error.

Is there a full migration guide from API3 to API4? I searched for one but could not find anything of the sort.

**Comment by shssoichiro** [comment]
@DJATOM It looks like these are the remaining fixes to get it to work: https://github.com/shssoichiro/vapoursynth-preview/commit/5e8eb1ff1f930068f49a98d48a6180319a331a55

**Comment by DJATOM** [comment]
Ah, indeed. I forgot to update for latest changes. Will make new patch soon.

**Comment by DJATOM** [comment]
Here comes [updated patch](https://github.com/DJATOM/vapoursynth-preview/commit/55b30f02cb72a09fe291b9839d4a701b81a8ac11).
